testing/python/collect.py

```diff
@@ -143,6 +143,18 @@ class test(object):
             "*collected 0*",
         ])
 
+    def test_staticmethid(self, testdir):
```

Comment:
Spelling fix: argh, yes. Underscores: happy to change it but I was copying the decorator name and the method just below named `test_setup_teardown_class_as_classmethod` (for the `@classmethod` decorator)